apps/emails-and-messages/src/lib/fetch-app-permissions.ts

```diff
@@ -0,0 +1,28 @@
+import { Client, gql } from "urql";
+import {
+  FetchAppPermissionsDocument,
+  FetchAppPermissionsQuery,
+  PermissionEnum,
+} from "../../generated/graphql";
+
+gql`
+  query FetchAppPermissions {
+    app {
+      permissions {
+        code
+      }
+    }
+  }
+`;
+
+export async function fetchAppPermissions(client: Client): Promise<PermissionEnum[]> {
+  const { error, data } = await client
+    .query<FetchAppPermissionsQuery>(FetchAppPermissionsDocument, {})
+    .toPromise();
+
+  if (error) {
+    return [];
```

Comment:
**question:** Should we not throw an error?

Reply:
it should, lets not hide errors